data/building-hours/6-5-tostrud-climbing-wall.yaml

```diff
@@ -4,10 +4,9 @@ category: Gym
 schedule:
   - title: Hours
     hours:
-      - {days: [Mo, Tu, We, Th], from: '6:30am', to: '10:45pm'}
-      - {days: [Fr], from: '6:30am', to: '8:45pm'}
-      - {days: [Sa], from: '9:00am', to: '8:45pm'}
-      - {days: [Su], from: '12:00pm', to: '10:45pm'}
+      - {days: [Mo, Tu, We, Th, Fr], from: '4:00pm', to: '9:00pm'}
```

Comment:
This needs to be
- Sunday–Thursday: 4pm–9pm
- Friday: 4pm–8pm

Reply:
It is

Reply:
We have an unofficial rule of thumb that the schedules start on Monday

Reply:
So I split his into M-Fr, Sa, Su